### DISCUSSION TITLE
Add a field "isResizing" to InteractionState in the onInteractionStateChange callback

### DISCUSSION THREAD

**Discussion by achen187** [uber/react-map-gl#1470] [Ideas]
I use isZooming and isDragging in onInteractionStateChange because I only make API calls if the user is not in the middle of a zoom or a drag.

I'd love to know whether the user is in the middle of resizing their window as well, so I only make an API call when they're done resizing the window.

**Comment by Pessimistress** [comment]
This library does not detect if the user is resizing their window. Map resize can be the result of window resize and/or CSS change/animation.

There is also no JavaScript API for "the user in the middle of resizing their window", as `resize` is a discrete event. You can listen to `onResize` and [debounce](https://www.npmjs.com/package/debounce) any API call from there.
